config/default/kustomization.yaml

```diff
@@ -1,12 +1,12 @@
 # Adds namespace to all resources.
-namespace: spanner-autoscaler
+namespace: spanner-autoscaler-system
 
 # Value of this field is prepended to the
 # names of all resources, e.g. a deployment named
 # "wordpress" becomes "alices-wordpress".
 # Note that it should also match with the prefix (text before '-') of the namespace
 # field above.
-# namePrefix: spanner-autoscaler-
+namePrefix: spanner-autoscaler-
```

Comment:
Is this change required?

Reply:
This was enabled by default. I think it is good because this makes it easier to identify the resources created for this controller specifically. If multiple operators/controllers are deployed in the same namespace, then this really helps in distinguishing resources quickly.